modules/dxp/apps/search-experiences/search-experiences-web/src/main/resources/META-INF/resources/sxp_blueprint_admin/js/shared/EditTitleModal.js

```diff
@@ -61,9 +61,11 @@ export default function EditTitleModal({
 
 	const defaultLocaleBCP47 = formatLocaleWithDashes(defaultLocale);
 
-	const [description, setDescription] = useState(initialDescription);
+	const [description_i18n, setDescription_i18n] = useState(
+		initialDescriptionI18n
+	);
 	const [hasError, setHasError] = useState(false);
-	const [title, setTitle] = useState(initialTitle);
+	const [title_i18n, setTitle_i18n] = useState(initialTitleI18n);
```

Comment:
We shouldn't mix naming conventions. Since states are camel-cased, this should be: 
- `[descriptionI18n, setDescriptionI18n]`
- `[titleI18n, setTitleI18n]`

This will also help with identifying the state being used versus a variable within a function.